index.rst

```diff
@@ -1,51 +1,212 @@
+
+To begin, edit the values near the top of the ``decam_process`` script to reflect your desired directory names
+for where processed images will live, as well as to specify the visits and CCDs you wish to process. The default presets are
+
+.. code-block:: python
+
+   repo = 'ingested_15A38/'  # used by ingest, ingestCalibs, processCcd
+   calibrepo = 'calibingested_15A38/'  # used by ingestCalibs, processCcd
+   processedrepo = 'processed_15A38/'  # used by processCcd, diffIm
+   diffimrepo = 'diffim_15A38_g/'  # used by diffIm
+   visits = [410927, 411033]  # used by processCcd, diffIm
+   # NOTE: visits assumes the first element is template and the rest are science
+   ccdnum = '1..62'  # used by processCcd, diffIm
+   # NOTE: the default '1..62' value includes all of the DECam CCDs
+
+In general, ``repo`` refers to where ingested images will live, ``calibrepo``
+refers to where ingested calibration products (flats and biases) will live, ``processedrepo`` refers to where "calexp"
+images will live (i.e., those that have been processed with ``processCcd`` including steps 1 through 3 in :ref:`Overview <Overview>`),
+and ``diffimrepo`` refers to where difference images and DIA Sources (catalogs) will ultimately live.
+Visit numbers can be found in image headers or retrieved from the registry database created in ``repo`` during image ingestion 
+(visit numbers are not used during ingestion, so you may set them after this step). The first visit you specify
+in the list will be used as the template.
+
+Finally, copy or link the Pan-STARRS astrometric reference catalog into the directory you've chosen for 
+``repo`` and call it ``ref_cats``. If you are working on the ``lsst-dev`` server, you can link the Gaia, 
+Pan-STARRS, and SDSS catalogs by 
+
+.. prompt:: bash
+   
+   mkdir repo
+   ln -s /datasets/refcats/htm/htm_baseline repo/ref_cats
+
+If you wish to use an astrometric reference catalog other than Pan-STARRS, you must update the code in the ``doProcessCcd``
+function of ``decam_process.py`` accordingly. It is not necessary to explicitly ``mkdir`` the other repositories.
+
+If you really don't want to deal with astrometric reference catalogs, you can skip the astrometry and 
+photometric calibration steps by editing the contents of ``args`` in the ``doProcessCcd`` function of ``decam_process.py``. 
+In this situation, you would set both ``calibrate.doAstrometry=False`` and ``calibrate.doPhotoCal=False``. 
+Be aware that the difference imaging will not work well in this case, however, because the visits 
+will not be precisely lined up to the same WCS.
+
+Once you are ready, run the following:
+
+1. Ingest the raw images
+
+.. prompt:: bash
+   
+   python decam_process.py ingest -f path/to/rawimages/
+   
+2. Ingest the calibration products (defects and fringes, if present, must be done manually)
+
+.. prompt:: bash
+
+   python decam_process.py ingestCalibs -f path/to/calibrations/
```

Comment:
This step is confusing. So the first line creates a script `ingestCalibs.py` in `calibrepo`?

Instead of cd-ing into `calibrepo`, can the script be run via:
```
./calibrepo/ingestCalibs.py repo --calib calibrepo --calibType defect --validity 999 ./path/do/defects
```
? Or even better, could the `ingestCalibs.py` script be run from `decam_process.py`?